server/hwapi/data_models/models.py

```diff
@@ -74,7 +78,7 @@ class Machine(Base):
 class Certificate(Base):
     __tablename__ = "certificate"
     created_at: Mapped[datetime] = mapped_column(nullable=False)
-    name: Mapped[str] = mapped_column(String(80), nullable=True)
+    name: Mapped[str] = mapped_column(String(80), unique=True, nullable=True)
```

Comment:
`Certificate.name` is the name of what? I'm assuming certificates don't have name. If it's the name of a machine, then perhaps we should rename this variable to `machine_name`. But at the same time, I'm not sure what use we have with machine name

Reply:
This is what we call a certificate name, it's like a canonical ID but for certificates

![image](https://github.com/canonical/hardware-api/assets/84857215/4aa9761b-33f9-433a-841f-131dff691685)

Reply:
Oh I see. Do we need this bit of info in `hardware-api`?